Docker: improve config files setup, refs #12721

jraddaoui opened the pull request:
- Don't overwrite existing config files.
- Use Memcached host and port from config.

Connects to #822.

sbreker reviewed (approved):
Looks great!  Thanks @jraddaoui!  👍

jraddaoui committed:
Docker: improve config files setup, refs #12721

- Don't overwrite existing config files.
- Use Memcached host and port from config.
